lib/web/terminal.go

```diff
@@ -1032,6 +1005,14 @@ func WithTerminalStreamFileTransferHandlers(fileTransferRequestC chan<- *session
 	}
 }
 
+func NewWStream(ws WSConn) *WsStream {
```

Comment:
```suggestion
func NewWStream(ws WSConn) *WSStream {
```

If we're going to use `WS` to mean "websocket" then lets consistently capitalize it.